src/Components/LandingPage/LandingPage.scss

```diff
@@ -9,3 +9,8 @@
 .expand-section {
     background-color: var(--pf-global--palette--white);
 }
+
+.sidebar-panel {
+    height: 500pt;
```

Comment:
should we rather use px here? :)
Dunno about the value, on my monitor 750px worked better, but I do not have a strong opinion :)